### ISSUE TITLE
Can't locate object method "sign_sha256" via package "Crypt::Perl::ECDSA::PrivateKey"

### ISSUE THREAD

**Issue by eserte** [brtastic/perl-bitcoin-crypto#3]
On some of my smokers t/13-PrivateKey.t fails:
```
An error occured in Bitcoin subroutines: Can't locate object method "sign_sha256" via package "Crypt::Perl::ECDSA::PrivateKey" at /usr/home/cpansand/.cpan/build/2021090509/Bitcoin-Crypto-1.000_01-6/blib/lib/Bitcoin/Crypto/Role/BasicKey.pm line 38. (raised at t/13-PrivateKey.t, line 38)# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 5.
t/13-PrivateKey.t ...... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 5 subtests passed 
```

**Comment by brtastic** [comment]
Thank you @eserte, can you confirm that the version of Crypt::Perl in that environment is 0.33 or lower?

**Comment by eserte** [comment]
Checked one of the failing smokers:
```
$ perl5.32.0 -MCrypt::Perl\ 9999
Crypt::Perl version 9999 required--this is only version 0.33.
```

**Comment by brtastic** [comment]
should be fixed in `1a4dbeda7df777f04b3ba761b121b16d570b3bb3`
